app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java

```diff
@@ -9,7 +9,6 @@
  * <h1>Kotlin Data class to parse data object in retrofit response from susi client.</h1>
  */
 public class Datum extends RealmObject {
-
```

Comment:
This is Java class

Reply:
Yes. This too can be converted to data class. Will be sending a PR to check for such classes and convert them to data class. I think one or two classes have still not been converted to data classes yet in susi response package.

Reply:
OK

Reply:
@iamareebjamal Converting this class to a Kotlin data class is leading to issues because there are a number of generated files that inherit this file. And, on compiling I get a number of errors. Should I modify the comment instead?

Reply:
OK